### ISSUE TITLE
Season 1 Website Expansion Plans

### ISSUE THREAD

**Issue by kempsterrrr** [Developer-DAO/developerdao.com#244]
@Nazeeh21 who else needs to be tagged in this?

Wanted to open a disucssion about plans for the website in Season 1.

Here is my understanding of what is currently on the roadmap that might not be finished before S1 starts:

- [ ] CMS Implementation with internationalisation and localisation

Some ideas for what we could create during Season 1:

- **/projects** - A curated grid view of active/delivered projects in the DAO 
- **./projects/project-name** - more detailed breakdown of active projects in the DAO
- **/events** - A list of publically accessible events the DAO is either running or partnering on
- **/dashbaord** - a data dashboard tracking DAO stats similar to [Forefonts DAO Terminal](https://www.terminal.co/community/featured/ffdao.eth)

Would love to get others thoughts here!! Sure there is some cool stuff we can do on the website with some many amazing developers :)

**Comment by kempsterrrr** [comment]
@mohamedsadiq @eknobl

**Comment by Nazeeh21** [comment]
The proposed ideas are good and will be helpful for anyone visiting the website. So I think that'd be a good decision to include those things.

**Comment by mohamedsadiq** [comment]
I agree with Nazeeh21, The proposed ideas will help us achieve one of the most important goals for Dao, Definitely worth considering!

**Comment by mhmulford0** [comment]
I see the site/cms as a place to aggregate a lot of the disparate elements that currently compose the DAO. I would suggest projects first. What projects are ongoing and where to find them was the first question i had upon joining.

I have experience with Strapi as a headless CMS solution. I saw someone else was already using one called Prismic, however it only offers a hosted solution and is not open source:  More info: https://docs.strapi.io/developer-docs/latest/getting-started/introduction.html. 

We should focus using open source software in the projects that we build. This is a project i would be very interesting in tackling.

**Comment by kempsterrrr** [comment]
> I see the site/cms as a place to aggregate a lot of the disparate elements that currently compose the DAO. I would suggest projects first. What projects are ongoing and where to find them was the first question i had upon joining.

Completely agree. It it could really help people with answering that questions. I also hope it would help people feel a bigger sense of reward for the projects they are working in the DAO that they are being showcased in this way.

> I have experience with Strapi as a headless CMS solution. I saw someone else was already using one called Prismic, however it only offers a hosted solution and is not open source: More info: https://docs.strapi.io/developer-docs/latest/getting-started/introduction.html.

> We should focus using open source software in the projects that we build. This is a project i would be very interesting in tackling.

Personally all for open source but do we really want to undo all the work @alanpcurrie has done? The decision to use that CMS was made out in open and I'd hate to do that to someone at this stage after the effort they have put in cc @Nazeeh21 @mohamedsadiq

**Comment by Nazeeh21** [comment]
I agree with Kemp. Alan is working on cms and has come a long way into it. It won't be a good decision to undo all the things as it'll waste all his time and effort. So i believe for now it'd be good to keep going with the prismic.

**Comment by Nazeeh21** [comment]
Yes, even I feel that after the comment from Alan on discord, It'd be good to move to strapi. As it also has many more add-on benefits to offer

**Comment by alanpcurrie** [comment]
happy to support, I did a bit of refactoring around the layout, and components to allow cms data to be used that might be useful for Strapi integration too

**Comment by kempsterrrr** [comment]
Awesome, you're a legend @alanpcurrie - @mhmulford0 over to you, can you share the work you've done so far on the strapi implementation?

As for thinking about the other suggested features at the top of this issue, how would people like to progress? @Nazeeh21 and @mohamedsadiq you did such an amazing job on the new website would be great to take a lead on how you approached it if you're up for it?

**Comment by mhmulford0** [comment]
@kempsterrrr https://github.com/mhmulford0/dd-site for the frontend of the projects. this is an all in on repo that is set up for storybook, testing, linting, Github actions, pre-commit hooks, etc

backend is here https://github.com/mhmulford0/dd-website-backend although this is just a basic strapi deploy with my initial ideas of what the data model looks like

**Comment by mhmulford0** [comment]
@Nazeeh21 that is exactly what I am thinking as well. The main landing page can be a place that points to the other projects.

**Comment by kempsterrrr** [comment]
@mhmulford0 thanks for sharing these links, haven't had a chance to have a look yet but will do soon.

> We can integrate the static landing page into the project's work, although I believe they can live separate. https://developerdao.com/ serves the same function as it does presently.

> And then something like https://projects.developerdao.com/ for the project work, https://terminal.developerdao.com/ for the terminal idea, etc. This also allows for work to be done reasonably independently and in parallel and should keep blockers to a minimum.

Agree with @Nazeeh21 before we build anything further on the FE we should lead with design. @mhmulford0 not sure I understand the value of hosting the projects on a subdomain vs `/projects` (grid view or whatever) and `/projects/project-name` for detailed static view of the project. What is the thought process? From a DAO marketing POV having, valuable content on a subdomain decreases the SEO juice we'd receive over time.

As for the terminal/data stats yeah can see that making sense living on a subdomain.

**Comment by mhmulford0** [comment]
Subdomain was the first idea, not super stuck on it. And if a subdirectory is better for SEO, all the better. I didn't consider SEO and didn't know that it doesn't like subdomains, good info. @kempsterrrr @0xBabatunde 

No hard design decisions made yet so all is well I believe. I was mostly just poking around, wiring up the things, etc. Any progress is good at this point! 

How do we go about getting the ball rolling on the design for the new elements? I saw the the landing page in The Figma but didn't see anything else.

**Comment by kempsterrrr** [comment]
Yeah, we good :) I was thinking about how we move forward last night. @Nazeeh21 @mohamedsadiq did a great job on this new version of the website so probs best to take their lead if they have the time?

I've spun up [space on the DAO miro](https://miro.com/app/board/uXjVOa4GdEE=/?moveToWidget=3458764529073911526&cot=14) for the website and will start dropping some structure and thoughts on there as I have time today.

**Comment by Nazeeh21** [comment]
Yes, i believe we can start by designing the projects page and like wise detailed projects page. @mohamedsadiq let us know if anything is required from our side for the projects section.

**Comment by kempsterrrr** [comment]
I'm going to be handing on on VC5 for a bit thinking about the website if anyone is around

**Comment by Nazeeh21** [comment]
Hey @kempsterrrr, I am not at home currently, so won't be able to join. I have dm'd @mohamedsadiq about this convo and will coordinate with him about the `projects` design

**Comment by kempsterrrr** [comment]
FYI, folks, @mhmulford0 has transferred the Strapi backend to our org here - https://github.com/Developer-DAO/dd-website-backend

**Comment by kempsterrrr** [comment]
@mohamedsadiq and I chatted today and he is onboard to move ahead with the design taking inspiration from the data model. We spoke about the `/projects` page being a grid of projects with maybe one featured one at the top and then the `projects/project-name` pages being like profiles with a card of details of the projects on the left and description etc. on the RHS

for anyone coming here we have another repo associated with this project which is a Strapi CMS over [here](https://github.com/Developer-DAO/cms) with the data model for projects being discussed in an issues there

@mhmulford0 strapi is now deployed on digital ocean and we are re-routing it to cms.developer.dao with a posgres backend
